pkgs/stdenv/generic/setup.sh

```diff
@@ -1495,17 +1495,7 @@ distPhase() {
 
 showPhaseHeader() {
     local phase="$1"
-    case "$phase" in
-        unpackPhase) echo "unpacking sources";;
-        patchPhase) echo "patching sources";;
-        configurePhase) echo "configuring";;
-        buildPhase) echo "building";;
-        checkPhase) echo "running tests";;
-        installPhase) echo "installing";;
-        fixupPhase) echo "post-installation fixup";;
-        installCheckPhase) echo "running install tests";;
-        *) echo "$phase";;
-    esac
+    echo "Running phase: $phase"
```

Comment:
I found an old nix ux doc(which eelco was involved in) in which `Running phase:` was used, so I changed it to that

https://github.com/tweag/nix-ux/blob/master/first_steps_with_nix_v2.md